client/driver/shutdownOracle.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
```

Comment:
Pass in the username/password here instead of hardcoding it (same with startupOracle.sh)

Reply:
I need to switch to the user space in which the oracle environment is installed (su - xxx). So I need to add an entry "username_of_system_with_oracle" in driver_config.json. And no username or password is required to start or shutdown the Oracle server.

Reply:
All other problems in your review are fixed exactly according to your comments. Thank you!